src/main/resources/org/sagebionetworks/web/client/widget/team/controller/TeamEditModalWidgetViewImpl.ui.xml

```diff
@@ -13,7 +13,7 @@
 		<b:ModalHeader closable="true">
 		    <b:Heading size="H4" text="Edit Team" addStyleNames="displayInline"/>
 		    <w:HelpWidget helpMarkdown="A team of users can be used to grant access to content for the whole group or give attribution to a team." 
```

Comment:
Teams are a way to easily manage groups of users for the purposes of project access, communication, and challenge participation.